src/es/hackstore/src/eu/kanade/tachiyomi/animeextension/es/hackstore/Hackstore.kt

```diff
@@ -195,22 +197,35 @@ class Hackstore : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
     override fun episodeFromElement(element: Element): SEpisode { throw UnsupportedOperationException() }
 
     // ============================ Video Links =============================
+    private fun extractUrlFromDonFunction(fullUrl: String): String {
+        val client = OkHttpClient()
```

Comment:
Is creating a new client really necessary? If it is, make it a class property instead so it won't create a new client each time the function is called.